main.html

```diff
@@ -0,0 +1,107 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0"> 
+    <title>Communet</title>
+    <link rel="stylesheet" href="style.css"> <!--to connect to the style.css file-->
+    <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/bootstrap@5.2.2/dist/css/bootstrap.min.css"/>
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.2.0/css/all.min.css"/>
+</head>
+<body>
+    <!-- home section start -->
+    <div class="home">
+        <h1>Communet</h1>
+    </div>
+    <!--about us page-->
+
+    <section class="about-us">
+        <div class="ab-container">
+            <div class="row">
+                <div class="col-sm-5">
+                    <h1 class="text-center about-us-h">About us</h1>
+                    <p class="about-us-p">Our idea is that you create a communet profile, users can apply to join, you reach economic agreements with them (shopping, chores, food, utilities, hobbies) and everyone agrees beforehand, and the system manages the expenses, reminds users of tasks, keeps track of who needs to do what, etc. (Ofc the money management part would be a mock up)
```

Comment:
The Communet platform allows people with shared interests and values to create a financially cooperative community, with pre-established norms and procedures. 

Chores, shopping, and bills are all pre-configured and agreed upon before co-habitation begins, taking the mess and complexity out of starting a shared life. 

Future chores, expenses, and savings can be added and voted on at any time.